test-cypress/integration/ttyg/chat-list.spec.js

```diff
@@ -129,9 +141,40 @@ describe.skip('TTYG chat list', () => {
         TTYGViewSteps.getChatByDayGroups().should('have.length', 1);
     });
 
+    // FIXME: Skipped because of exception thrown by GDB
```

Comment:
All TTYG UI functionality is tested with mocked backend because it is using an external service (the OpenAI API) which we can't rely on to be present always for our tests. That's why you should mock the export API for these tests as well.